### PR TITLE
fix: invalid urls and typos 

### LINKED ISSUE #506: Invalid URLs in this repo
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
> 
> /kind enhancement

/kind bug

**What happened**:
There are several invalid URLs in this repo:

```bash
A number of unique URLs to check: 43
Error: https:/docs.openshift.org/latest/install_config/install/prerequisites.html in ../go/src/kubevirt.io/kubevirt-ansible/playbooks/README.md
Error: https:/docs.openshift.org/latest/install_config/install/advanced_install.html in ../go/src/kubevirt.io/kubevirt-ansible/playbooks/README.md
Error: https:/docs.openshift.org/latest/install_config/install/planning.html in ../go/src/kubevirt.io/kubevirt-ansible/playbooks/README.md
Error: https:/docs.openshift.org/latest/install_config/persistent_storage/index.html in ../go/src/kubevirt.io/kubevirt-ansible/playbooks/README.md
Error: https:/docs.openshift.org/latest/install_config/install/host_preparation.html in ../go/src/kubevirt.io/kubevirt-ansible/playbooks/README.md
Error: http:/jenkins.ovirt.org/blue/organizations/jenkins/kubevirt_kubevirt-ansible_standard-check-pr/activity in ../go/src/kubevirt.io/kubevirt-ansible/automation/README.md
Error: https:/git.k8s.io/community/contributors/guide/release-notes.md in ../go/src/kubevirt.io/kubevirt-ansible/.github/PULL_REQUEST_TEMPLATE.md
Error: https:/openshift.org/ in ../go/src/kubevirt.io/kubevirt-ansible/CONTRIBUTING.md
Error: http:/docs.ansible.com/ansible/latest/playbooks_best_practices.html in ../go/src/kubevirt.io/kubevirt-ansible/CONTRIBUTING.md
Error: https:/github.com/kubevirt/kubevirt/releases/download in ../go/src/kubevirt.io/kubevirt-ansible/roles/kubevirt/README.md
Error: https:/groups.google.com/forum/ in ../go/src/kubevirt.io/kubevirt-ansible/CONTRIBUTING.md
A number of invalid URLs: 11
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- kubevirt-ansible version(use `git rev-parse HEAD`):
- KubeVirt version (use `virtctl version`):
- OpenShift/Kubernetes version (use `kubectl version` or `oc versions`):
- VM or VMI specifications:
- Ansible version  (use `ansible --version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by adityaramteke**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #506 

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: playbooks/README.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ No additional cni plugin will be deployed.
 ### SELinux
 
 In case you are experiencing permission or SELinux issues, please consider
-creating an issue for [kubevirt](https://github.com/kubevirt/kubevirt/issues/) 
+creating an issue for [kubevirt](https://github.com/kubevirt/kubevirt/issues/)
```
Comment (gouyang) [inline playbooks/README.md]: Not sure if it need the space here, I think no.

File: roles/kubevirt/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Deploy KubeVirt resources onto a cluster.
 |cluster|openshift |<ul><li>openshift</li><li>kubernetes</li></ul>|Cluster type.|
 |namespace|kube-system | |Namespace to create resources.|
 |apb_action|provision| <ul><li>provision</li><li>deprovision</li></ul>|Action to perform.|
-|release_manifest_url|https://github.com/kubevirt/kubevirt/releases/download|||
+|release_manifest_url|https://github.com/kubevirt/kubevirt/releasese|||
```
Comment (gouyang) [inline roles/kubevirt/README.md]: `releasese` should be `releases`.

File: playbooks/README.md (hunk 1 above)
Comment (adityaramteke) [reply playbooks/README.md]: Exactly, we don't need any space here.

File: roles/kubevirt/README.md (hunk 2 above)
Comment (adityaramteke) [reply roles/kubevirt/README.md]: Thank you for this. let me update it.

Comment (adityaramteke) [reply roles/kubevirt/README.md]: done

File: playbooks/README.md (hunk 1 above)
Comment (gouyang) [reply playbooks/README.md]: :( sorry, I mean it need a space here. Just forget type "not".

Comment (adityaramteke) [reply playbooks/README.md]: I think it does not require any space `an issue for kubevirt and report what is `
I have checked in my forked repo.

Comment (adityaramteke) [reply playbooks/README.md]: you may have a look here https://github.com/adityaramteke/kubevirt-ansible/blob/invalide_url/playbooks/README.md#selinux

**Comment by gouyang** [comment]
LGTM

**Pull request merged**